packages/core/content-manager/admin/src/preview/utils/previewScript.ts

```diff
@@ -56,34 +56,79 @@ const previewScript = (shouldRun = true) => {
    * Functionality pieces
    * ---------------------------------------------------------------------------------------------*/
 
-  const setupStegaDecoding = async () => {
+  const setupStegaDOMObserver = async () => {
     if (DISABLE_STEGA_DECODING) {
       return;
     }
 
-    const { vercelStegaDecode: stegaDecode } = await import(
+    const { vercelStegaDecode: stegaDecode, vercelStegaClean: stegaClean } = await import(
       // @ts-expect-error it's not a local dependency
       // eslint-disable-next-line import/no-unresolved
       'https://cdn.jsdelivr.net/npm/@vercel/stega@0.1.2/+esm'
     );
 
-    const allElements = document.querySelectorAll('*');
+    const processElementForStega = (element: Element) => {
```

Comment:
Maybe something like: encodeElement or applySteganographyToElement would be more explicit?


```suggestion
    const encodeElement = (element: Element) => {
```

or

```suggestion
    const applySteganographyToElement = (element: Element) => {
```